cmd_generate.go

```diff
@@ -56,6 +57,7 @@ func NewGenerateCommand(l remoteTemplateLoader) *GenerateCommand {
 		"run `gowrap template list` for details")
 	fs.Var(&gc.vars, "v", "a key-value pair to parametrize the template,\narguments without an equal sign are treated as a bool values,\ni.e. -v foo=bar -v disableChecks")
 	fs.StringVar(&gc.localPrefix, "l", "", "put imports beginning with this string after 3rd-party packages; comma-separated list")
+	fs.StringVar(&gc.ignoreMethod, "ignore_method", "", "ignore method")
```

Comment:
Let's make it boolean flag. Because I don't think people will be picking methods to ignore, most likely these will be just all unexported methods to ignore.

```go
fs.BoolVar(&gc.ignoreUnexported, "u", false, "ignore unexported methods")
```